AppCenter/AppCenterTests/MSDeadLockTests.m

```diff
@@ -40,10 +40,15 @@ - (NSString *)groupId {
 }
 
 - (void)channel:(id<MSChannelProtocol>)channel didPrepareLog:(id<MSLog>)log internalId:(NSString *)internalId flags:(MSFlags)flags {
+  (void)channel;
+  (void)log;
+  (void)internalId;
+  (void)flags;
```

Comment:
We are using __unused in many other places.

Reply:
+1 for `__unused` in parameters.